test/jdk/java/awt/Scrollbar/ScrollbarMouseWheelTest/ScrollbarMouseWheelTest.java

```diff
@@ -108,7 +107,9 @@ public void test() {
         testComp(sb2, true);
         // down on sb2
         testComp(sb2, false);
+
         frame.dispose();
+
         System.out.println("Test done.");
         if (sb1upevents == REPS &&
```

Comment:
Note that all these flags are updated on one thread and used on another.

Reply:
Updated, also added extra delay after test.